zh_cn/news/_posts/2014-11-13-ruby-2-1-5-is-released.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: news_post
+title: "Ruby 2.1.5 发布了"
+author: "nagachika"
+translator: "Will"
+date: 2015-05-28 12:00:00 +0000
```

Comment:
@Airmacho

Note that the original date must be kept. Fixed with ff3c284bfb54792ae0f8ab82c7bcbd34a2d85e3d.